app/controllers/application_controller.rb

```diff
@@ -1,5 +1,12 @@
+require "./lib/jsonapi/responder"
```

Comment:
Would it be good to add this `/lib/jsonapi` to autoload path in `application.rb`?